packages/video_player/video_player/example/lib/main.dart

```diff
@@ -55,7 +55,24 @@ class _App extends StatelessWidget {
         ),
         body: TabBarView(
           children: <Widget>[
-            _BumbleBeeRemoteVideo(),
+            SingleChildScrollView(
+              child: Column(
+                children: const <_BumbleBeeRemoteVideo>[
+                  _BumbleBeeRemoteVideo(
+                    title: 'With remote mp4 and asset close caption',
+                    videoUrl:
+                        'https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4',
+                    useAssetCC: true,
+                  ),
+                  _BumbleBeeRemoteVideo(
+                    title: 'With remote mp4 and embedded close caption',
+                    videoUrl:
+                        'https://mtoczko.github.io/hls-test-streams/test-vtt-fmp4-segments/playlist.m3u8',
```

Comment:
can you use flutter website to host those assets? (you can find examples in this file